FitGoal/FitGoal/Singup&Singin/CustomUIViews/AuthenticationFormView.swift

```diff
@@ -72,26 +103,37 @@ class AuthenticationFormView: UIStackView, UITextFieldDelegate {
 }
 
 private class CustomTextField: UITextField {
+    let textFieldType: TextFieldType
     
-    let buttonLine: UIView = {
+    private let buttonLine: UIView = {
         let line = UIView()
         line.backgroundColor = UIColor(red: 0.9, green: 0.9, blue: 0.9, alpha: 1)
         return line
     }()
     
-    convenience init(placeholder: String) {
-        self.init(frame: .zero)
-        self.attributedPlaceholder = placeholder.formattedText(
+    let textFieldMessageLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .right
+        label.isHidden = true
```

Comment:
Do you need to set this value to `true` here? I would expect it to match the state of the error message.